build.gradle

```diff
@@ -177,57 +177,3 @@ task buildImage(type: Exec) {
         return standardOutput.toString()
     }
 }
-
-task buildTrustedImage(type: Exec) {
```

Comment:
It might worth it to call docker/sconify.sh here

Reply:
Do you mean keep a `buildTrustedImage` task calling `docker/sconify.sh` ?

Reply:
Yes :+1:

Reply:
done in 246d87d

Reply:
Very nice, thank you for your time :pray: